sync-conflict.ftd

```diff
@@ -2,11 +2,11 @@
 
 -- ft.h1: What is an FPM Sync Conflict?
 
-FPM can handle most [sync](glossary/sync/#sync) operations on its own with
+FPM can handle most [sync](glossary/#sync) operations on its own with
 automatic merging features. A sync conflict is an event that occurs when FPM is
 unable to automatically resolve differences in the file between latest
-[version](glossary/sync/#version) available on the [server](glossary/sync/#server-project)
-and the changes done against [client](glossary/sync/#client-project)'s latest
+[version](glossary/#version) available on the [server](glossary/#server-project)
+and the changes done against [client](glossary/#client-project)'s latest
```

Comment:
Linking like this is so tedious. We need the term feature! I thought of one more syntax, `[term:remote]`. And if you want different text to appear, `[different text](term:remote)`.